test/integration_test/common_test.go

```diff
@@ -124,18 +128,24 @@ func setup() error {
 
 	}
 	logrus.Infof("Using provisioner: %s", provisioner)
-	if err = schedulerDriver.Init(scheduler.InitOptions{
-		SpecDir:             "specs",
+	var customAppConfig map[string]scheduler.AppConfig
+	// For non-PX backups use default-scheduler in apps instead of stork
+	if provisioner != "pxd" {
```

Comment:
Same as the default backup location should put this in the driver in a separate change

Reply:
will send separate PR in torpedo.